cita-executor/core/src/cita_executive.rs

```diff
@@ -786,6 +793,41 @@ impl Default for EnvInfo {
     }
 }
 
+// There is not reverted expcetion in VMError, so handle this in ExecutedException.
+#[derive(Debug)]
+pub enum ExecutedException {
+    VM(VMError),
+    NativeContract(NativeError),
+    Reverted,
+}
+
+impl Error for ExecutedException {}
+
+impl fmt::Display for ExecutedException {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        let printable = match *self {
+            ExecutedException::VM(ref err) => format!("exception in vm: {:?}", err),
+            ExecutedException::NativeContract(ref err) => {
+                format!("exception in native contract: {:?}", err)
+            }
+            ExecutedException::Reverted => "execution reverted".to_owned(),
+        };
+        write!(f, "{}", printable)
+    }
+}
+
+impl From<VMError> for ExecutedException {
+    fn from(err: VMError) -> Self {
+        ExecutedException::VM(err)
+    }
+}
+
+impl From<NativeError> for ExecutedException {
+    fn from(err: NativeError) -> Self {
+        ExecutedException::NativeContract(err)
+    }
+}
+
```

Comment:
Put all error handling logic in another seperated file is better.

Reply:
This is some errors about `executive`, I will think about it later!